### PR TITLE
Fix sphinx-lint errors

### PR DESCRIPTION
**Pull request by szepeviktor**
### brief description of changes

Ran `sphinx-lint --enable all --max-line-length 718 docs/`
The longest line is 718 long.
This is the result.
```
No problems found.
```

#### issues addressed
[sphinx-lint](https://github.com/sphinx-contrib/sphinx-lint) errors

---

This box **must** be checked
- [x] the PR branch was originally made from the base branch

This box **should** be checked
- [x] after these changes the docs build locally without error

This box **should only** be checked you intend to follow through on it (we can do it on our end too)
- [ ] I will `cherry-pick` all commits in this PR into other branches that should have them _after_ this PR is merged

### CODE REVIEW

File: convert_html_help_to_rst.py [hunk 1]
```diff
@@ -29,13 +30,8 @@ def convert_html(file_path):
                 if line.rstrip() == "":
                     continue
                 outfile.write(line)
-            
-    
-    print output
-    
+
+    print(output)
```
Comment (ekansa) [inline convert_html_help_to_rst.py:34]: Wow! Py2 was in this?? Nice catch!

File: docs/examples/arches/function.py [hunk 2]
```diff
@@ -31,8 +31,8 @@ def get_class_module(self):
             if module != None:
                 break
         if import_success == False:
-            print 'Failed to import ' + mod_path
-            print import_error
+            print('Failed to import ' + mod_path)
```
Comment (ekansa) [inline docs/examples/arches/function.py:34]: Again, thanks for catching Python 2 syntax.

**Review by ekansa — APPROVED** [review]
Thank you! These are very helpful improvements!

File: docs/examples/arches/function.py (hunk 2 above)
Comment (szepeviktor) [reply docs/examples/arches/function.py:34]: I've seen your usernamename in https://github.com/robots.txt

**Comment by szepeviktor** [comment]
Glad to contribute.

File: docs/examples/arches/function.py (hunk 2 above)
Comment (ekansa) [reply docs/examples/arches/function.py:34]: Yes, @szepeviktor an ill considered experiment some years ago. Earned me a little infamy!

**Pull request merged**